src/modal/modal.js

```diff
@@ -828,3 +828,11 @@ angular.module('ui.bootstrap.modal', ['ui.bootstrap.multiMap', 'ui.bootstrap.sta
 
     return $modalProvider;
   });
+
+var CNTRL_REG = /^(\S+)(\s+as\s+([\w$]+))?$/;
+function identifierForController(controller) {
```

Comment:
This function and variable should be inside the service it is being used in - this will expose these as globals I believe after the build process.